agent/nrpe.rb

```diff
@@ -50,8 +51,8 @@ class Nrpe<RPC::Agent
       # Example :
       #          plugin.nrpe.conf_dir = /etc/nagios/nrpe
       #          plugin.nrpe.conf_file = checks.nrpe
-      def self.run(command)
-        nrpe_command = Nrpe.plugin_for_command(command)
+      def self.run(command, *args)
```

Comment:
This use of splatting and unsplatting is quite strange.  Wouldn't it make more sense to make the signature of Nrpe.run be:

```
def self.run(command, args = [])
```

Then in the codepaths where the methods wasn't called with the optional second it would be equivalent to supplying an empty array.